docs/release_notes.rst

```diff
@@ -0,0 +1,71 @@
+Release notes
+=============
+
+2.2.0
+-----
+
+Bodhi 2.2.0 is a security and feature release, with a few bug fixes as well.
+
+
+Security
+^^^^^^^^
+
+This update addresses `CVE-2016-1000008 <https://github.com/fedora-infra/bodhi/pull/857>`_ by
+disallowing the re-use of solved captchas. Additionally, the captcha is
+`warped <https://github.com/fedora-infra/bodhi/commit/f0122855>`_ to make it more difficult to
+solve through automation.
```

Comment:
Add a credit here for Patrick

https://github.com/distributedweaknessfiling/DWF-Database/blob/master/DWF-Database-2016.csv#L17